content/guides/author-libraries.md

```diff
@@ -129,6 +130,42 @@ This means that your library expects a dependency named `lodash` to be available
 
 If your libary targets UMD, it's important to add all of the above mentioned ways of loading the external (`commonjs`, `commonjs2`, `amd` and `root`) as leaving one out will cause strange errors for a consumer trying to load your library in that environment.
```

Comment:
There's actually a typo to fix here. `libary` -> `library`.

Maybe fix that while at it. I'll merge then.